src/sdk/pynni/nni/compression/torch/compressor.py

```diff
@@ -143,6 +196,56 @@ def _expand_config_op_types(self, config):
                 expanded_op_types.append(op_type)
         return expanded_op_types
 
+class PrunerModuleWrapper(torch.nn.Module):
+    def __init__(self, module, module_name, module_type, config, pruner):
+        """
+        Wrap an module to enable data parallel, forward method customization and buffer registeration.
+
+        Parameters
+        ----------
+        module : pytorch module
+            the module user wants to compress
+        config : dict
+            the configurations that users specify for compression
+        module_name : str
+            the name of the module to compress, wrapper module shares same name
+        module_type : str
+            the type of the module to compress
+        pruner ： Pruner
+            the pruner used to calculate mask
+        """
+        super().__init__()
+        # origin layer information
+        self.module = module
+        self.name = module_name
+        self.type = module_type
+        # config and pruner
+        self.config = config
+        self.pruner = pruner
+
+        # register buffer for mask
+        self.register_buffer("weight_mask", torch.ones(self.module.weight.shape))
+        if hasattr(self.module, 'bias') and self.module.bias is not None:
+            self.register_buffer("bias_mask", torch.ones(self.module.bias.shape))
+
+        # register user specified buffer
+        self.buffer_kwargs = {}
```

Comment:
suggest to change the name to `registered_buffers`, because there is no relation between this variable and kwargs, kwargs is how `calc_mask` receives input arguments.

Reply:
fixed.